test/test.js

```diff
@@ -34,6 +34,13 @@ specify("It can be configured with package.json", function (done) {
     testOutputMatches("test3", done, undefined, { dontTransform: true });
 });
 
+specify("It can be configured with the runtimePath option", function (done) {
+    testOutputMatches("test4", done, {
+        self: true,
+        runtimePath: 'jade/runtime'
```

Comment:
Since this is just the default, it doesn't actually test anything. Maybe make a copy of the runtime file in test/stuff/test4/custom-runtime.js and pass in that path?